CONTRIBUTING.rst

```diff
@@ -40,7 +40,7 @@ commits that show up in ``git blame`` 10 years from now make more sense.
 Coding Guidelines
 =================
 
-**Strongly prefer Unix timestamps** over ``datetime`` structures and always
+**Strongly prefer Unix timestamps or datetime objects in UTC** and always
```

Comment:
and that means using pytz, what has been only needed when converting from unix timestamp to datetime, but not the other way around, because there was not functionality in datetime for that.
However, working on UTC when possible, it's right.